### ISSUE TITLE
introduce proxy settings to exclude certain hosts or networks

### ISSUE THREAD

**Issue by TP75** [nextcloud/desktop#1583]
### Expected behaviour
Available feature (both by GUI and by CLI config) for adjusting the **proxy settings to exclude certain hosts or networks**. The NC desktop ‘_Network window - specify the proxy manually_' shall provide an **appopriate string input** field.

### Actual behaviour
Generally, the system proxy setup should ensure to ignore all local addresses including but not limited to e.g. a local NC server on the same machine. However, one may choose to specify the proxy manually or cannot make use of the system proxy for other reasons.

Obviously, the NC desktop ‘Network window - specify the proxy manually’ option lacks such feature for adjusting the proxy settings to exclude certain hosts or networks.

See the NC user forum for an use case example and report:
https://help.nextcloud.com/t/windows-client-use-a-proxy-only-for-one-connection/63136/4?u=tp75

### Steps to reproduce

Quite obvoius a lack of featrue.

### Client configuration
Client version: **2.6.1 RC1 and earlier**

Operating system: **all**

**Comment by TP75** [comment]
still **open** for _2.6.1stable_ unfortunately

**Comment by urbanware-org** [comment]
I agree with the contributor of this issue. This would be a useful feature. :+1:

**Comment by joshtrichards** [comment]
Related: #5608

**Comment by Rello** [comment]
Hello,

this ticket is pretty old and the client had several fundamental redesigns.
If the issue still exists, please create a new ticket with current information.

Thank you
